### PR TITLE
[PT-BR] translations of app index.ts and nav.ts

### PR DESCRIPTION
**Pull request by reynnan**

### CODE REVIEW

File: packages/typescriptlang-org/src/copy/pt/index.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  index_standards_copy: `
+  <p>O time do TypeScript contribui para o comitê TC39 que ajudam a guiar a evolução do Javascript.</p>
+
+  <p>Quando as novas propostas atingem o estágio 3, eles estão prontos para inclusão no TypeScript.</p>
+  
+  <p>Por exemplo, a equipe do TypeScript defendeu propostas como Encadeamento opcional, Operador de coalescência nula, Throw Expressions e RegExp Match Indices.</p>`,
+
+  index_gradual: "Adoção Gradual",
+  index_gradual_copy: `
+<p>Adotar TypeScript não é uma escolha binária, você pode começar anotando o JavaScript existente com JSDoc,  e então trocar alguns arquivos para serem checados através do TypeScript e com o tempo preparar todo seu código para ser convertido completamente</p>
```
Comment (danilofuchs) [inline packages/typescriptlang-org/src/copy/pt/index.ts]:
```suggestion
<p>Adotar TypeScript não é uma escolha binária, você pode começar anotando o JavaScript existente com JSDoc, e então trocar alguns arquivos para serem checados através do TypeScript e com o tempo preparar todo seu código para ser convertido completamente</p>
```

File: packages/typescriptlang-org/src/copy/pt/index.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+  index_started_title: "Iniciando",
+  index_started_handbook: "Manual",
+  index_started_handbook_blurb: "Aprenda a linguagem",
+  index_started_docs: "Comece um Projeto",
+  index_started_docs_blurb: "Encontre uma ferramenta de bootstrapping",
+  index_started_tooling: "Ferramentas",
+  index_started_tooling_blurb: "Aprofunde-se no TypeScript",
+  index_started_community: "Community",
+  index_started_community_blurb: "Keep up to date",
```
Comment (danilofuchs) [inline packages/typescriptlang-org/src/copy/pt/index.ts]: Faltou uma tradução

**Review by danilofuchs — CHANGES_REQUESTED** [review]

File: packages/typescriptlang-org/src/copy/pt/index.ts (hunk 2 above)
Comment (reynnan) [reply packages/typescriptlang-org/src/copy/pt/index.ts]: Tradução adicionada: b101efa

File: packages/typescriptlang-org/src/copy/pt/index.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    "Qualquer navegador, Qualquer SO, em qualquer lugar onde Javascript rodar. Completamente código aberto.",
+
+  index_cta_play: "Tente no seu navegador",
+  index_cta_install: "Instale localmente",
+
+  index_what_is: "O que é TypeScript?",
+
+  index_what_is_js: "JavaScript e Mais",
+  index_what_is_js_copy: `
+<p>TypeScript é uma linguagem código aberto construida emcima do Javascript, uma das ferramentas mais usadas do mundo, adicionando definições de tipagem estáticas</p>
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/index.ts]:
```suggestion
<p>TypeScript é uma linguagem código aberto construida em cima do Javascript, uma das ferramentas mais usadas do mundo, adicionando definições de tipagem estáticas</p>
```

File: packages/typescriptlang-org/src/copy/pt/index.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+<p>Tipos fornecem um jeito de descrever a froma de um objeto, fornecendo melhor documentação, e permitindo que o TypeScript valide se seu código está funcionando corretamente.</p>
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/index.ts]:
```suggestion
<p>Tipos fornecem um jeito de descrever a forma de um objeto, fornecendo melhor documentação, e permitindo que o TypeScript valide se seu código está funcionando corretamente.</p>
```

File: packages/typescriptlang-org/src/copy/pt/nav.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
+export const navCopy = {
+  skip_to_content: "Pule para o conteúdo principal",
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/nav.ts]:
```suggestion
  skip_to_content: "Pular para o conteúdo principal",
```

File: packages/typescriptlang-org/src/copy/pt/nav.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  nav_documentation_short: "Docs",
+  nav_download: "Baixar",
+  nav_community: "Comunidade",
+  nav_playground: "Playground",
+  nav_playground_short: "Play",
+  nav_handbook: "Manual",
+  nav_tools: "Ferramentas",
+  nav_search_placeholder: "Pesquise nas Docs",
+  nav_search_aria: "Pesquise o site TypeScript",
+  nav_beta: "Pesquise o site TypeScript",
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/nav.ts]:
```suggestion
  nav_beta: "Pesquise no site do TypeScript",
```

File: packages/typescriptlang-org/src/copy/pt/nav.ts [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  nav_documentation: "Documentação",
+  nav_documentation_short: "Docs",
+  nav_download: "Baixar",
+  nav_community: "Comunidade",
+  nav_playground: "Playground",
+  nav_playground_short: "Play",
+  nav_handbook: "Manual",
+  nav_tools: "Ferramentas",
+  nav_search_placeholder: "Pesquise nas Docs",
+  nav_search_aria: "Pesquise o site TypeScript",
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/nav.ts]:
```suggestion
  nav_search_aria: "Pesquise no site do TypeScript",
```

File: packages/typescriptlang-org/src/copy/pt/nav.ts [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  nav_beta: "Pesquise o site TypeScript",
+  nav_beta_notification:
+    "Nota: nós estamos testando a versão v2 do site durante esse final de semana, se esquecemos de algo qual você confia, por favor <a>crie uma issue em microsoft/TypeScript-Website</a>.",
+  // let me know if you can't make this work in your lang:
+  // TypeScript X.Y [stable][between?]Z.Y[beta/rc]
+  nav_version_stable_prefix: "está disponível agora",
+  nav_version_between: ", ",
+  nav_version_beta_prefix: "está atualmente em beta.",
+  nav_version_rc_prefix: "tem um candidato a lançamento para tentar.",
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/nav.ts]:
```suggestion
  nav_version_rc_prefix: "tem um candidato a lançamento para testar.",
```

File: packages/typescriptlang-org/src/copy/pt/nav.ts [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+  nav_this_page_in_your_lang: "Essa página está disponível na sua língua",
+  nav_this_page_in_your_lang_open: "Aberto",
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/nav.ts]:
```suggestion
  nav_this_page_in_your_lang_open: "Navegar",
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Changes in words

**Comment by orta** [comment]
The windows fail is my fault, I've pushed a fix to v2 for it https://github.com/microsoft/TypeScript-Website/commit/03b93553f603bb03e00e4ec65e4c394bd5654018 (the filename had a space in it) - the fact that translators weren't pinged is fixed in https://github.com/microsoft/TypeScript-Website/commit/4c9d2573e9255235feb37044b8cd3262e1cd8562 too (I added tests in https://github.com/orta/code-owner-self-merge/pull/4 )

**Comment by orta** [comment]
I'll merge this, it looks like all feedback is addressed - thanks all!

**Pull request merged**